frontend/packages/policy-editor/src/components/PolicySummary/PolicySummary.tsx

```diff
@@ -14,6 +16,14 @@ export function PolicySummary(): React.ReactElement {
       <StudioHeading spacing={true} level={4} size='xs'>
         {t('policy_editor.summary_heading')}
       </StudioHeading>
+      <StudioAlert severity='info' title='Tilbakemelding' className={classes.feedbackAlert}>
```

Comment:
Should we use the translation file here too? 😊 

```suggestion
      <StudioAlert severity='info' title={t('nb-json-key')} className={classes.feedbackAlert}>
```